main/accounts/index.ts

```diff
@@ -541,14 +541,24 @@ export class Accounts extends EventEmitter {
         return cb(new Error('Signer unavailable'))
       }
     }
+
+    if (signer && signer.status !== 'ok') {
+      const crumb = { view: 'expandedSigner', data: { signer: signer.id } }
+      store.navDash(crumb)
+      return cb(new Error('Signer unavailable'))
```

Comment:
On reflection this approach is better than trying to bundle the hot signer logic with the hardware since the former does not need the lastSignerType so we would be making an unnecessary store access

Reply:
yeah, the only reason I changed this in the first place is because the hot signer didn't work, even when unlocked they would show as unavailable

Reply:
Oh ok, I thought I'd checked them.  Good change in any case